cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -20,15 +20,18 @@
     },
     "MinSize": {
       "Description": "Initial number of compute EC2 instances / vCpus to launch within the cluster.",
-      "Type": "Number"
+      "Type": "Number",
+      "Default": 0
     },
     "DesiredSize": {
       "Description": "Desired number of compute EC2 instances / vCpus to launch within the cluster.",
-      "Type": "Number"
+      "Type": "Number",
+      "Default": 0
     },
     "MaxSize": {
       "Description": "Maximum number of compute EC2 instances / vCpus to launch within the cluster.",
-      "Type": "Number"
+      "Type": "Number",
+      "Default": 0
```

Comment:
Are we sure we want to put the default of MaxSize to 0? I'd put 10.
Then I'd add this change to the commit message and to the PR description.

Reply:
This is for HIT clusters, which does not longer rely on this parameter. Default for SIT cluster is set in mappings and it's still 10.